maven/bnd-export-plugin/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,6 @@
+eclipse.preferences.version=1
```

Comment:
Make sure the .settings folder has the same content as the other maven projects. We want to keep them all in sync.